### PR TITLE
[1700] Add currency in doc

### PR DESCRIPTION
**Pull request by yyyutong**

### CODE REVIEW

File: specifications/factoring/factoring.yml [hunk 1]
```diff
@@ -879,6 +879,10 @@ components:
                     description: Percentage to finance of the claim total amount
                     minimum: 0
                     example: 90
+              currency:
+                type: string
+                description: Currency for the business terms in ISO 4217 format
+                example: SEK
```
Comment (FabriSilve) [inline specifications/factoring/factoring.yml:885]: I don't know if it actually change the position but can you put this field before the lineItems one?
I think we need also to be more explicit about what this currency is about :thinking: 
Can you update the message is:
> Currency of the line items in ISO 4217 format. if not specified, will be asserted that the currency of the line items is the same of the account.

File: specifications/factoring/factoring.yml [hunk 2]
```diff
@@ -1451,6 +1455,10 @@ components:
                     type: string
                     description: The type of the fee
                     example: reminder
+            currency:
+              type: string
+              description: Currency for the business terms in ISO 4217 format
```
Comment (FabriSilve) [inline specifications/factoring/factoring.yml:1460]: > Currency of the financement in ISO 4217 format

File: specifications/factoring/factoring.yml [hunk 3]
```diff
@@ -1524,6 +1532,10 @@ components:
               type: number
               description: The number of days the claim full payment is overdue. If the number is negative, the due date of the payments is still in the future. If the number is positive, the claim is now late.
               example: 5
+            currency:
+              type: string
+              description: Currency for the business terms in ISO 4217 format
```
Comment (FabriSilve) [inline specifications/factoring/factoring.yml:1537]: > Currency for claim in ISO 4217 format

**Review by FabriSilve — CHANGES_REQUESTED** [review]

File: specifications/factoring/factoring.yml [hunk 4]
```diff
@@ -1524,6 +1532,12 @@ components:
               type: number
               description: The number of days the claim full payment is overdue. If the number is negative, the due date of the payments is still in the future. If the number is positive, the claim is now late.
               example: 5
+            currency:
+              type: string
+              description: Currency for claim in ISO 4217 format
+
+
```
Comment (FabriSilve) [inline specifications/factoring/factoring.yml:1539]: You have a lot of empty lines here :grin:

Comment (yyyutong) [reply specifications/factoring/factoring.yml:1539]: :scream:

**Review by BlackEydPanda — APPROVED** [review]

**Pull request merged**